### DISCUSSION TITLE
Using Dreamcast without P1

### DISCUSSION THREAD

**Discussion by Downcraft** [darthcloud/BlueRetro#55] [Q&A] [answered]
Hello, 

first of all thank you for sharing this awesome project!
I am using universal firmware and setting everything to DC with web configuration. So everythting is working perfectly as expected. But removing P1 cable will lead to endless pairing request from all controllers. 

Is there a way to use P2-P4 with blueretro and using P1 with original controller so I have memory/saving (VMU) possibility while using 3 wireless controllers for the other players? Or is this not possible because of the way maple bus is working? I thought it might be somehow possible to pair 4 controllers with blueretro and disconnecting/replacing p1 cable and remain with functioning p2-p4.

**Comment by darthcloud** [comment, marked as answer]
Always use the first plugs for BlueRetro, in your case BR P1 -> DC P2, BR P2 -> DC P3 & BR P3 -> DC P4, Leave BR P4 unconnected.

**Reply by Downcraft** [reply]
Thank you for this very fast reply. Sadly this is not working for me. If plugging it the way you mentioned and trying to play multiplayer game dreamcast does not regocnize any controller on Dreamcast Port 2. If I plug in all 4 Br cable all 4 controllers are regocnized by dreamcast. Using original controller + br will lead to only regocnizing original controller

**Reply by Downcraft** [reply]
If plugging in br1->dc p1, br2 -> dc p2 , br3 -> dc p3, br4 -> unconnected it won't pair either so I have to setup new wiring like described if p4 is not wired?

**Reply by darthcloud** [reply]
This is weird P4 should be safe to leave unplug, you got latest firmware? Try put P4 pins (IO26, IO27) to GND?

**Reply by Downcraft** [reply]
I put them to ground and everything works now. 
I use Firmware v0.19 should be the latest or am I wrong?

**Reply by Downcraft** [reply]
Thank you very much for your help! Now my setup is near perfect when ble series x ctrls is working too in future.😇

**Reply by darthcloud** [reply]
v0.19 is good, the pins got weak pull-up on them that should avoid this but maybe the unused cable is stronger than the pull-up and create false interrupt that make the watchdog trigger.

So now it work if you connect them as a suggested in original answer?

**Reply by Downcraft** [reply]
Yes those are some long wires so this might be the case, because sometimes I got pairing success but not for long term. 

Yes with IO26/27 pulled to 3.3V the described setup of yours work and I can combine BR with original controller. Thank you for your Support.

**Reply by darthcloud** [reply]
Hey thanks for the tips! Really appreciated :)
